includes/class-newspack-newsletters.php

```diff
@@ -706,22 +707,40 @@ public static function api_set_mailchimp_interest( $request ) {
 				);
 			}
 
-			$segment_opts = ( 'no_interests' === $request['interest_id'] ) ?
-				(object) [] :
-				[
+			$has_interest  = $interest_id_param && ! $is_unsetting_interest;
+			$tag_ids_param = $request['tag_ids'];
+
+			$segment_opts = (object) [];
+
+			if ( $has_interest || $tag_ids_param ) {
+				$segment_opts = [
 					'match'      => 'any',
-					'conditions' => [
-						[
-							'condition_type' => 'Interests',
-							'field'          => $field,
-							'op'             => 'interestcontains',
-							'value'          => [
-								$interest_id,
-							],
-						],
-					],
+					'conditions' => [],
 				];
 
+				if ( $has_interest ) {
+					$segment_opts['conditions'][] = [
+						'condition_type' => 'Interests',
+						'field'          => $field,
+						'op'             => 'interestcontains',
+						'value'          => [
+							$interest_id,
+						],
+					];
+				}
+
+				if ( $tag_ids_param ) {
+					foreach ( $tag_ids_param as $tag_id ) {
+						$segment_opts['conditions'][] = [
+							'condition_type' => 'StaticSegment',
+							'field'          => 'static_segment',
+							'op'             => 'static_is',
+							'value'          => $tag_id,
+						];
+					}
+				}
+			}
```

Comment:
If a Tag segment is specified, the plugin will activate the Send button even if there are 0 recipients. Ideally the campaign can only be sent if the segmentation logic includes at least one subscriber, and the pre-send modal should indicate the recipient count. Here are screenshots showing the disconnect between the segmentation in Mailchimp vs. the plugin:

<img width="1512" alt="Screen Shot 2020-07-05 at 2 52 17 PM" src="https://user-images.githubusercontent.com/1477002/86539992-35e61400-becf-11ea-95d4-4048021c3787.png">
<img width="1512" alt="Screen Shot 2020-07-05 at 2 52 12 PM" src="https://user-images.githubusercontent.com/1477002/86539995-3aaac800-becf-11ea-847d-29aa4d707e1c.png">

Reply:
Fixed to display the actual number of recipients in caa29470a09f267b67896e74a7945f86997dadd9. I've also updated it so only tags with recipients are suggested in the tags field (0fc84692fd0fd23c77cc663f5c19293d33071c81).